projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -523,4 +523,21 @@ public function add_settings_page_notice() {
 
 		add_action( 'admin_notices', $admin_notices );
 	}
+
+	/**
+	 * Update Jetpack menu for Atomic.
+	 */
+	public function add_jetpack_menu() {
+		parent::add_jetpack_menu();
+
+		// When the interface is set to wp-admin, we need to link Jetpack stats to Calypso stats.
+		if ( get_option( 'wpcom_admin_interface' ) === 'wp-admin' ) {
+			$this->update_submenus(
+				'jetpack',
+				array(
+					'stats' => 'https://wordpress.com/stats/day/' . $this->domain,
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)

I'm wondering if we should use `/wp-admin/admin.php?page=stats` for the URL. 
The current implementation (using Calypso stats `/stats/day/`) seems to kind of diverge from the Core behavior, which uses `/wp-admin/admin.php?page=stats` for Jetpack stats. 
Maybe I'm missing context; please correct me if I'm wrong! 🙂

Reply:
We also had a discussion about it here: p1705963764843679-slack-CKZHG0QCR

We decided to pause it since it is not on our project's scope.

Reply:
Thanks for letting me know. I caught up 🙂